jetty-server/src/main/java/org/eclipse/jetty/server/LowResourceMonitor.java

```diff
@@ -95,14 +88,79 @@ public void run()
             if (isRunning())
             {
                 monitor();
-                _scheduler.schedule(_monitor,_period,TimeUnit.MILLISECONDS);
+                _scheduler.schedule( _monitor, _period, TimeUnit.MILLISECONDS);
             }
         }
     };
 
+    private boolean _monitorThreads=true;
+    private int _maxConnections;
+    private long _maxMemory;
+
     public LowResourceMonitor(@Name("server") Server server)
     {
-        _server=server;
+        _server = server;
+
+        // create default LowResourceChecks..
+        if(_monitorThreads){
+            this._lowResourceChecks.add( new MainThreadPoolLowResourceCheck() );
```

Comment:
Can this be done in the constructor? how can monitorThreads be set before the constructor?  Perhaps create these default check instances either in doStart or in the setters?

Reply:
ooops will fix